### PR TITLE
System purge

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR starts a big System context rebuilt in anticipation of `Inbox`.

In this PR we:
 1. Transform System aggregates into projections and post events instead of commands.
 2. Strip system entities of their power to dispatch (or deny dispatching of) events and commands to non-event-sourced domain entities.

Currently, idempotency tests are disabled, since there are no more idempotency checks except for `IdempotencyGuard` for aggregates.

The System context receives non-handled events (e.g. `EventCausedCommand`). Those will be required later for integration with tracing services.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+<!-- 
+ 
+This file was generated using the Gradle `generatePom` task. 
+This file is not suitable for `maven` build tasks. It only describes the first-level dependencies of 
+all modules and does not describe the project structure per-subproject.
+ 
+ -->
+
+<groupId>io.spine</groupId>
```
Comment (armiol) [inline pom.xml]: I wonder how come this file hasn't been added to VCS previously.

File: server/src/main/java/io/spine/server/DefaultRepositoryString.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+package io.spine.server;
+
+import io.spine.annotation.Internal;
+import io.spine.server.entity.model.EntityClass;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.lang.String.format;
+
+/**
+ * Handles the string representation of the {@linkplain DefaultRepository default repositories}.
```
Comment (armiol) [inline server/src/main/java/io/spine/server/DefaultRepositoryString.java]: Let's discuss this concept.

File: server/src/main/java/io/spine/system/server/EntityLogRepository.java [hunk 3]
```diff
@@ -20,29 +20,38 @@
...
 
 import static io.spine.server.route.EventRoute.withId;
 
 /**
- * The repository for {@link EntityHistoryAggregate}s.
+ * The repository for {@link EntityLogProjection}s.
  */
-final class EntityHistoryRepository
-        extends SystemRepository<EntityHistoryId, EntityHistoryAggregate> {
+final class EntityLogRepository
```
Comment (armiol) [inline server/src/main/java/io/spine/system/server/EntityLogRepository.java]: I think this one should die at all.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

**Pull request merged**